tests/system_test.py

```diff
@@ -430,7 +431,11 @@ def __init__(self, name=None, listen_port=None, wait=True,
         self.args = [sys.executable, os.path.join(os.path.dirname(os.path.abspath(__file__)), self.server_file)]
         if self.cl_args:
             self.args += self.cl_args
-        super(Http2Server, self).__init__(self.args, name=name, expect=expect, **kwargs)
+        server_env = dict(os.environ)
```

Comment:
Was wondering if this line is necessary for the tests to pass 
```
server_env = dict(os.environ)
```

Reply:
When passing an environment to a Process, the sub-process's entire environment is replaced.  Since we still need things like PATH we need to pass the parent's environment down. This patch creates a new environment based on the parent's env + the extra config vars. By doing a copy of the env we leave the parent's env unchanged.

Does that answer your question?